ibm_catalog.json

```diff
@@ -154,6 +154,194 @@
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "Create an IBM Cloud Object Storage instance",
+                    "description": "Create an IBM Cloud Object Storage instance."
+                },
+                {
+                    "title": "Create an IBM Cloud Object Storage bucket",
+                    "description": "Create either a cross-regional or regional Object Storage bucket."
+                },
+                {
+                    "title": "Secure access with CBR",
+                    "description": "Create context-based restriction (CBR) rules for both instances and buckets."
+                },
+                {
+                    "title": "Attach access tags",
+                    "description": "Attach access tags to the IBM Cloud Object Storage instance."
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Instance",
+                    "name": "instance",
+                    "working_directory": "solutions/instance",
+                    "compliance": {
+                        "authority": "scc-v3",
+                        "profiles": [
+                            {
+                                "profile_name": "IBM Cloud Framework for Financial Services",
+                                "profile_version": "1.5.0"
+                            }
+                        ]
+                    },
+                    "architecture": {
+                        "features": [
+                            {
+                                "title": "IBM Cloud Object Storage Instance",
+                                "description": "Create an IBM Cloud Object Storage instance."
+                            },
+                            {
+                                "title": "Create an IBM resource key",
+                                "description": "Create a resource key for the IBM Cloud Object Storage instance."
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "IBM Cloud Object Storage Solution",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/reference-architectures/instance.svg",
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "This architecture supports creating and configuring an IBM Cloud Object Storage instance."
+                            }
+                        ]
+                    }
+                },
+                {
+                    "label": "Secure Cross Regional Bucket",
+                    "name": "secure-cross-regional-bucket",
+                    "working_directory": "solutions/secure-cross-regional-bucket",
```

Comment:
This is an extension type DA, so you need to add the `install_type` and `dependencies` section to the JSON. 

```
         "install_type": "extension",
          "dependencies": [
            {
              "flavors": [
                "instance"
              ],
              "id": "68921490-2778-4930-ac6d-bae7be6cd958",
              "name": "deploy-arch-ibm-cos",
              "version": ">= v1.0.0"
            }
          ]
```